package.json

```diff
@@ -9,7 +9,7 @@
     "flow": "flow",
     "gen-snapshot-tests": "node utils/gen-snapshot-tests.js && prettier --write packages/*/generated-snapshot.test.js",
     "jest": "SNAPSHOT_INLINE_APHRODITE=1 jest",
-    "test": "npm run flow && npm run gen-snapshot-tests && npm run jest",
+    "test": "npm run lint && npm run gen-snapshot-tests && npm run flow && npm run jest",
```

Comment:
May as well run `lint` as part of `test.  I've also move `flow` to come after `gee-snapshots-test` in order to catch any changes.

Reply:
Great, thank you!